### DISCUSSION TITLE
Wallpaper from YouTube

### DISCUSSION THREAD

**Discussion by Besdroxk** [rocksdanister/lively#2386] [Ideas]
This is partly a question and an idea at the same time ;)

I liked Music Tunel, but I wanted the video to play. There is an option to put a link to the video as a web wallpaper, but then the music is also played + I want to take into account the change of the video. I haven't fully explored the Lively Wallpaper api yet, but I want to ask - **is it possible to open an http server using the api?**

In the browser where I listen to YouTube, I use Tampermonkey to write links to videos to localhost. And if it is possible to create an http server for a specific localhost, I could implement the intended logic + mute the video.

**Comment by rocksdanister** [comment]
I don't follow exactly what you are trying to do.

>  Lively Wallpaper api yet, but I want to ask - is it possible to open an http server using the api?

No, but you can create your own and load the address.

Music Tunnel is a webpage project I made using threejs, the music albumart is sent from Lively and it is loaded as a texture. You can load any texture you want in the javascript code.

**Reply by Colou1** [reply]
> You can load any texture you want in the javascript code.

Could you make an tutorial how to do that?

**Reply by brkeejp** [reply]
pretty sure you can start searching from here https://threejs.org/docs/#VideoTexture and change code a little bit.

**Comment by ginglebells6** [comment]
i did it but for some reason i cant type
